cli/tests/test_alias.rs

```diff
@@ -267,12 +269,7 @@ fn test_alias_global_args_in_definition() {
 fn test_alias_invalid_definition() {
     let test_env = TestEnvironment::default();
 
-    test_env.add_config(
-        r#"[aliases]
-    non-list = 5
-    non-string-list = [0]
-    "#,
-    );
+    test_env.add_config(r#"aliases.non-list = 5"#);
```

Comment:
These changes are necessary as we are eagerly evaluating aliases, so it would return it based on whichever alias was evaluated first, not the one that we try and use.